api/api.go

```diff
@@ -409,3 +411,7 @@ func (a *API) CipherDecryptStream(reader io.Reader, contentType string) ([]byte,
 func (a *API) CipherDecryptJSON(version byte, nonce, ciphertext []byte, algorithm string) ([]byte, error) {
 	return cipherDecryptFunc(a.source, cipher.ModeJSON, nil, "", version, nonce, ciphertext, algorithm)
 }
+
+func (a *API) GetSystemStatus() (*reqres.StatusResponse, error) {
+	return health.GetSystemStatus(context.Background(), a.source)
```

Comment:
None of the other API methods take a context argument.

What makes this one different?